app/models/tag.rb

```diff
@@ -40,6 +106,18 @@ def delete_and_update!
         tag_name:       name
       }
     )
-    destroy
+
+    # Create the delete activity.
+    repository.create_activity(
+      :delete,
+      owner:      actor,
+      recipient:  self,
+      parameters: {
+        repository_name: repository.name,
+        namespace_id:    repository.namespace.id,
```

Comment:
This can change in the future because I expect users will request to be able to delete also a namespace and all their contents. Deleting a namespace would cause this activity to break.